Library/UIAlertController.swift

```diff
@@ -6,6 +6,23 @@ import KsApi
 
 public extension UIAlertController {
 
+  public static func `init`(title: String,
```

Comment:
Should we name this consistently with the static function below? Something like:

`public static func alertController(with title: String, message: String?...)`